scripts/import/dbSNP_v2/update_new_sets.pl

```diff
@@ -0,0 +1,342 @@
+#!/usr/bin/env perl
+use strict;
+use DBI;
+use Socket;
+use Bio::EnsEMBL::Registry;
+use Getopt::Long;
+use POSIX qw(strftime);
+use Cwd qw(cwd);
+
+$| = 1;
+socketpair(CHILD, PARENT, AF_UNIX, SOCK_STREAM, PF_UNSPEC) or die "ERROR: Failed to open socketpair: $!";
+PARENT->autoflush(1);
+
+my $registry = 'Bio::EnsEMBL::Registry';
+my $config = configure();
+
+debug($config, "Importing variation set starting now");
+
+my $reg_file = $config->{registry};
+$registry->load_all($reg_file);
+my $db_adaptor = $registry->get_DBAdaptor("homo_sapiens", "variation");
+my $dbh = $db_adaptor->dbc;
+my $dbname;
+$dbname = $dbh->dbname;
+debug($config, "Connected to the new database $dbname");
+
+my $TMP_DIR = $config->{tmp};
+
+my $TMP_FILE = "variation_feature.txt";
```

Comment:
You could rename the variable name. As `variation_feature.txt` contains the new variation_feature you could name the variable `$new_vf_file`.